src/app/containers/manuscript/authors-info-details/index.tsx

```diff
@@ -77,7 +77,11 @@ const AuthorInfo: React.FC<AuthorInfoProps> = React.memo(({ author }) => {
         <div className={classes.authorInfoLine}>
           <img src={OrcidIcon} alt="orcid icon" className={classes.orcidIcon} /> 0000-0001-5000-0007
         </div>
-        <div className={classes.authorInfoLine}> No Competing interest declared </div>
+        <div className={classes.authorInfoLine}>
+          {author.hasCompetingInterest
+            ? `Competing interest: ${author.competingInterestStatement}`
```

Comment:
As a result of the issue noted below, I see...
![Screenshot 2020-07-06 at 13 31 56](https://user-images.githubusercontent.com/52923935/86593426-1ad9d980-bf8d-11ea-80e0-7f0bb746df8a.png)
But this should resolve if you sort out the 'hasCompetingInterest' flag.